pages/pipelines/environment_variables.md.erb

```diff
@@ -406,6 +406,13 @@ The repository of your pipeline. This variable can be set by exporting the envir
 
 Example: `"git@github.com:acme-inc/my-project.git"`
 
+<h3 class="h3-caps">BUILDKITE_REPO_MIRROR</h3>
+
+Only available if the experimental feature [git-mirrors](/docs/agent/v3#experimental-features-git-mirrors) is enabled.
```

Comment:
IMO it lacks the standard description element "This value can be set by doing X".

Reply:
```suggestion
Only available if the experimental feature [git-mirrors](/docs/agent/v3#experimental-features-git-mirrors) is enabled. The value cannot be modified.
```